requests_ntlm2/connection.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import re
 import socket
+import select
```

Comment:
travis build is failing at flake8 linting step because this line is in wrong alphabetic sort order for imports

Reply:
Seems to be solved now.